scheduler/planner.go

```diff
@@ -231,6 +235,10 @@ func (unit *Unit) RankValue() int64 {
 		// give generators a boost so people don't have to wait twice.
 		priority = priority * unit.distro.GetGenerateTaskFactor()
 	}
+	if stepbackTask {
+		// give stepback tasks a priority over mainline
+		priority = priority * unit.distro.GetStepbackTaskFactor()
```

Comment:
should this be `+=`? I saw that the generate tasks was the only one to not use the `+=` so I copied that

Reply:
We use += farther down because we're adding it to cachedValue specifically; here we're just modifying the base priority that we'll be using in the below calculations.